### PR TITLE
Fix corrections

### PR DESCRIPTION
**Pull request by lavanyavirushan**
fixed header and footer for HTML
corrected minor typos within site
Logo has link to the home page 
Social media is linked 
fixed text colour in about us page

### CODE REVIEW

**Review by nnavg — APPROVED** [review]
corrections look good.

File: assets/html/about.html [hunk 1]
```diff
@@ -1,176 +1,178 @@
...
+              >
+              <a
+                class="inline-block rounded-lg py-1 px-2 text-sm text-slate-700 hover:bg-emerald-100 hover:text-emerald-900"
+                href="./workout.html"
+                >Workouts</a
+              >
+              <a
+                class="inline-block rounded-lg py-1 px-2 text-sm text-slate-700 hover:bg-emerald-100 hover:text-emerald-900"
+                href="./diet.html"
+                >Mealplans</a
```
Comment (bambolelooo) [inline assets/html/about.html]: Meal plans

**Review by DexterDick — APPROVED** [review]
looks good

**Review by bambolelooo — APPROVED** [review]
Great

**Pull request merged**